Source/WebKit/ModelProcess/cocoa/ModelProcessModelPlayerProxy.mm

```diff
@@ -702,6 +739,22 @@ static CGFloat effectivePointsPerMeter(CALayer *caLayer)
         return;
 
     m_stageModeOperation = stagemodeOp;
+    applyStageModeOperationToDriver();
+}
+
+void ModelProcessModelPlayerProxy::applyStageModeOperationToDriver()
+{
+    switch (m_stageModeOperation) {
+    case WebCore::StageModeOperation::Orbit: {
+        [m_stageModeInteractionDriver operationDidUpdate:WKStageModeOperationOrbit];
+        break;
+    }
+
+    default: {
```

Comment:
Prefer listing out all enum values over using `default`. This ensures that in the future, people think about how new values should behave when adding them.